pattern_import_export/tests/common.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Akretion France (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from ..models.ir_exports import COLUMN_M2M_SEPARATOR
+from odoo.addons.queue_job.tests.common import JobMixin
+
+
+class ExportPatternCommon(JobMixin):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.Exports = cls.env['ir.exports']
+        cls.ExportsLine = cls.env['ir.exports.line']
+        cls.ExportsTab = cls.env['ir.exports.select.tab']
+        cls.Company = cls.env['res.company']
+        cls.Attachment = cls.env['ir.attachment']
+        cls.ExportPatternWizard = cls.env['export.pattern.wizard']
+        cls.Users = cls.env['res.users']
+        cls.Partner = cls.env['res.partner']
+        country_code_field = cls.env.ref("base.field_res_country__code")
+        country_model = cls.env.ref("base.model_res_country")
+        company_model = cls.env.ref("base.model_res_company")
+        cls.company1 = cls.env.ref('base.main_company')
+        company_name_field = cls.env.ref("base.field_res_company__name")
+        exports_vals = {"name": "Partner list", "resource": "res.partner"}
+        cls.separator = COLUMN_M2M_SEPARATOR
+        cls.ir_exports = cls.Exports.create(exports_vals)
```

Comment:
After thinking twice I saw that we start to create several ir.export, maybe it can be simpler to add then as demo data?

Reply:
I can do it in one shot in #9 (to avoid big conflicts) as I'm adding again some exports.